### ISSUE TITLE
Any planning to move parts to QGIS python/core?

### ISSUE THREAD

**Issue by rduivenvoorde** [boundlessgeo/lib-qgis-commons#10]
Hi, 

I think 

https://github.com/boundlessgeo/lib-qgis-commons/blob/master/qgiscommons2/network/networkaccessmanager.py

is a perfect candidate to move to QGIS itself... AND make other plugin authors use this class instead of the qgsnetworkmanager?

**Comment by volaya** [comment]
Yes, we are open to that, and once we get something stable, it would be great.

We have recently released in in pypi (that's the reason for the recent package renaming, since there were conflicts after a small refactoring we made), but it would be ideal to move it to core, indeed
